README.md

```diff
@@ -48,18 +51,23 @@ To **resume uploading** a dataset whose upload was interrupted:
 
 `cleanlab dataset upload -f [dataset filepath] --id [dataset ID]`
 
-A dataset ID is generated and printed to the terminal the first time the dataset is uploaded. It can also be accessed by
-visiting https://app.cleanlab.ai/datasets and selecting 'Resume' for the relevant dataset.
+A dataset ID is generated and printed to the terminal the first time the dataset is uploaded. It can also be accessed from the Datasets section of the Cleanlab Studio dashboard by visiting https://app.cleanlab.ai/ and selecting 'Resume' for the relevant dataset.
 
 #### Generate dataset schema
 
 To generate a dataset schema (prior to uploading your dataset):
 
 `cleanlab dataset schema generate -f [dataset filepath]`
 
+* For `Modality (text, tabular, image): ` enter one of the following:
+  * `text` - to only find label errors based on a single column of text in your dataset.
+  * `tabular` - to find data and label issues based on any subset of the column features.
```

Comment:
Is there a specific reason `tabular` says: "to find data and label issues" whereas text/image say: "to find label errors" ?

Probably should standardize the terminology unless there's specific reason not to. Eg. in this case I'd say: "to find and fix label errors" for all 3